Product/SoapUI_Test/RegressionSuite/Standard/DocQuerytestsforUDDI/DocQuerytestsforUDDI-soapui-project.xml

```diff
@@ -5447,7 +5447,7 @@ count(//ns6:AdhocQueryResponse[1]/ns3:RegistryObjectList[1]/ns3:ExtrinsicObject)
         </con:property>
       </con:properties>
       <con:reportParameters/>
-      <con:tearDownScript>nhinc.FileUtils.restoreToMasterConfiguration(context, log);</con:tearDownScript>
+      <con:tearDownScript>nhinc.FileUtils.restoreConfiguration(context.findProperty("GatewayPropDir"), log);</con:tearDownScript>
```

Comment:
Does it make any difference after changing function name in Teardown script   for this test case ? , because this test case is disabled originally.